package.json

```diff
@@ -149,10 +152,11 @@
     "package-hash": "^1.1.0",
     "pkg-conf": "^2.0.0",
     "plur": "^2.0.0",
-    "power-assert-context-formatter": "^1.0.4",
-    "power-assert-renderer-assertion": "^1.0.1",
-    "power-assert-renderer-succinct": "^1.0.1",
+    "pretty-format": "github:vadimdemedes/pretty-format#colors",
```

Comment:
I will publish it as `@ava/pretty-format` and resubmit my syntax highlighting proposal to facebook/jest (original at https://github.com/thejameskyle/pretty-format/issues/57).